### ISSUE TITLE
command "modifyboundary()" doesn't work properly

### ISSUE THREAD

**Issue by robizz** [hpfem/agros2d#79]
hi, i tried this simple script with Agros 2D 2.0.2.1173 under windows 7 x64
# model

newdocument("unnamed", "planar", "heat", 1, 2, "disabled", 1, 1, 0, "steadystate", 1, 1, 0)
# boundaries

addboundary("adiabatico", "heat_heat_flux", 0, 0, 0)
addboundary("interno", "heat_heat_flux", 0, 7.69, 20)
addboundary("esterno", "heat_heat_flux", 0, 25, 0)
# materials

addmaterial("isolante", 0, 0.04, 0, 0)
addmaterial("laterizio", 0, 0.25, 0, 0)
addmaterial("calcestruzzo", 0, 2, 0, 0)
# edges

addedge(-1.0836, 0, 0, 0, 0, "interno")
addedge(0, 0, 0.3, 0, 0, "interno")
addedge(0.3, 0, 1.3836, 0, 0, "interno")
addedge(1.3836, 0, 1.3836, 0.2, 0, "adiabatico")
addedge(1.3836, 0.2, 1.3836, 0.3612, 0, "adiabatico")
addedge(1.3836, 0.3612, 0.3, 0.3612, 0, "esterno")
addedge(0.3, 0.3612, 0, 0.3612, 0, "esterno")
addedge(0, 0.3612, -1.0836, 0.3612, 0, "esterno")
addedge(-1.0836, 0.3612, -1.0836, 0.2, 0, "adiabatico")
addedge(-1.0836, 0.2, -1.0836, 0, 0, "adiabatico")
addedge(0.3, 0, 0.3, 0.2, 0, "none")
addedge(0.3, 0.2, 0.3, 0.3612, 0, "none")
addedge(0, 0, 0, 0.2, 0, "none")
addedge(0, 0.2, 0, 0.3612, 0, "none")
addedge(-1.0836, 0.2, 0, 0.2, 0, "none")
addedge(0.3, 0.2, 1.3836, 0.2, 0, "none")
# labels

addlabel(-0.1, 0.201, 0, 0, "isolante")
addlabel(-0.1, 0.1, 0, 0, "laterizio")
addlabel(0.001, 0.001, 0, 0, "calcestruzzo")
addlabel(0.301, 0.201, 0, 0, "isolante")
addlabel(0.301, 0.1, 0, 0, "laterizio")

solve()
print surfaceintegral(0)['F']

modifyboundary("interno", "heat_heat_flux", 0, 4, 20)
solve()
print surfaceintegral(0)['F']

but both prints show the same value -6.4492

although two separate script without modifyboundary() print two different values (-6.4492 vs -6.46363) modifying directly the boundary creation with addboundary("interno", "heat_heat_flux", 0, 7.69, 20) to addboundary("interno", "heat_heat_flux", 0, 4, 20)

in conclusion, it seems that the command modifyboundary() has no effects on the boundary previously added.

Best Regards
